File: 프로그래머스/unrated/133499. 옹알이 （2）/옹알이 （2）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        String temp = "";
        for(int i=0; i<babbling.length; i++){
            temp = babbling[i].replace("aya","1").replace("ye","2").replace("woo","3").replace("ma","4");
            if(temp.contains("11") || temp.contains("22") || temp.contains("33") || temp.contains("44"))
                continue;
            else{
                temp = temp.replaceAll("[0-9]","");
                if(temp.length()==0) answer++;
            }
        }
        return answer;
    }
}

File: 프로그래머스/unrated/133499. 옹알이 （2）/옹알이 （2）.js
function solution(babbling) {
    var answer = 0;
    for(var i=0; i<babbling.length; i++){
        var str = babbling[i].replaceAll('aya', 1).replaceAll('ye', 2).replaceAll('woo',3).replaceAll('ma',4)
        console.log(str);
        if(!/[a-z]/.test(str) && str.length) {
            if(!str.includes('11')&&!str.includes('22')&&!str.includes('33')&&!str.includes('44')) answer++;
        }
    }
    return answer;
}